ccl/backupccl: TestBackupRestoreRandomDataRoundtrips failed

ccl/backupccl.TestBackupRestoreRandomDataRoundtrips [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3924617&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3924617&tab=artifacts#/) on master @ [dd7fd4e5a371ce4437fce383b9674cba2e212fe6](https://github.com/cockroachdb/cockroach/commits/dd7fd4e5a371ce4437fce383b9674cba2e212fe6):


```
=== RUN   TestBackupRestoreRandomDataRoundtrips
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestBackupRestoreRandomDataRoundtrips3997182020
    test_log_scope.go:80: use -show-logs to present logs inline
    panic.go:1038: -- test log scope end --
    panic.go:1038: geos: no locations to init GEOS
        goroutine 4216 [running]:
        runtime/debug.Stack()
        	GOROOT/src/runtime/debug/stack.go:24 +0x65
        github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func1()
        	github.com/cockroachdb/cockroach/pkg/util/leaktest/leaktest.go:110 +0x11f
        panic({0x4149640, 0xc00131d160})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc000f402d0, {0x5b135e0, 0xc00007e038})
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:551 +0x325
        panic({0x4149640, 0xc00131d160})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/sql/randgen.encodeInvertedIndexHistogramUpperBounds(0x80e9dc0, {0x5c9c1f8, 0xc00292cff0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:368 +0x467
        github.com/cockroachdb/cockroach/pkg/sql/randgen.randHistogram(0x5a97120, 0x80e9dc0)
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:301 +0x14c
        github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator.func1(0xc00345b080)
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:220 +0x92
        github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator(0xc0012690b0, {0xc001ce0570, 0x3, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:251 +0xa4d
        github.com/cockroachdb/cockroach/pkg/sql/randgen.MultiStatementMutation.Mutate(0x90, 0x44f7895, {0xc001ce0570, 0x8e, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:97 +0x2e
        github.com/cockroachdb/cockroach/pkg/sql/randgen.RandCreateTables(0x14, {0x44f7895, 0x5}, 0x3, {0xc001153640, 0x3, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/schema.go:85 +0x157
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTablesN(0xc001153670, 0xc0011536a0)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:89 +0x187
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTables(0x33841ed)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:76 +0x33
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.wrapCommonSetup.func1(0x3f0e9a0)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:46 +0x22
        github.com/cockroachdb/cockroach/pkg/ccl/backupccl.TestBackupRestoreRandomDataRoundtrips(0xc002039a00)
        	github.com/cockroachdb/cockroach/pkg/ccl/backupccl/backup_rand_test.go:53 +0x2cc
        testing.tRunner(0xc002039a00, 0x4ba2390)
        	GOROOT/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	GOROOT/src/testing/testing.go:1306 +0x35a
--- FAIL: TestBackupRestoreRandomDataRoundtrips (0.17s)
```
<details><summary>Help</summary>
<p>
See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
/cc @cockroachdb/bulk-io
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreRandomDataRoundtrips.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreRandomDataRoundtrips [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3975781&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3975781&tab=artifacts#/) on master @ [cadf1bf18001d00fc03f32f22d4e18b920e583ba](https://github.com/cockroachdb/cockroach/commits/cadf1bf18001d00fc03f32f22d4e18b920e583ba):


```
=== RUN   TestBackupRestoreRandomDataRoundtrips
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestBackupRestoreRandomDataRoundtrips2566827024
    test_log_scope.go:80: use -show-logs to present logs inline
    panic.go:1038: -- test log scope end --
    panic.go:1038: geos: no locations to init GEOS
        goroutine 4109 [running]:
        runtime/debug.Stack()
        	GOROOT/src/runtime/debug/stack.go:24 +0x65
        github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func1()
        	github.com/cockroachdb/cockroach/pkg/util/leaktest/leaktest.go:110 +0x11f
        panic({0x3fe4a60, 0xc004882680})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc00090a3f0, {0x557bd60, 0xc000140020})
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:548 +0x325
        panic({0x3fe4a60, 0xc004882680})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/sql/randgen.encodeInvertedIndexHistogramUpperBounds(0x7a6eee0, {0x5651598, 0xc0030d4150})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:368 +0x467
        github.com/cockroachdb/cockroach/pkg/sql/randgen.randHistogram(0x54ff4e0, 0x7a6eee0)
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:301 +0x14c
        github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator.func1(0xc004a5d200)
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:220 +0x92
        github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator(0xc003b8e930, {0xc0031de000, 0x4, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:251 +0xa4d
        github.com/cockroachdb/cockroach/pkg/sql/randgen.MultiStatementMutation.Mutate(0x90, 0x439452e, {0xc0031de000, 0x8e, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:97 +0x2e
        github.com/cockroachdb/cockroach/pkg/sql/randgen.RandCreateTables(0x14, {0x439452e, 0x5}, 0x4, {0xc005bb3640, 0x3, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/schema.go:85 +0x157
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTablesN(0xc005bb3670, 0xc005bb36a0)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:90 +0x187
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTables(0x320302d)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:77 +0x33
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.wrapCommonSetup.func1(0x3da98e0)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:47 +0x22
        github.com/cockroachdb/cockroach/pkg/ccl/backupccl.TestBackupRestoreRandomDataRoundtrips(0xc0022ff1e0)
        	github.com/cockroachdb/cockroach/pkg/ccl/backupccl/backup_rand_test.go:53 +0x2cc
        testing.tRunner(0xc0022ff1e0, 0x45fb9c8)
        	GOROOT/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	GOROOT/src/testing/testing.go:1306 +0x35a
--- FAIL: TestBackupRestoreRandomDataRoundtrips (0.16s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreRandomDataRoundtrips.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreRandomDataRoundtrips [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4004727&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4004727&tab=artifacts#/) on master @ [01c0c739dd83b3e662d715404f2193d747f4fb9e](https://github.com/cockroachdb/cockroach/commits/01c0c739dd83b3e662d715404f2193d747f4fb9e):


```
=== RUN   TestBackupRestoreRandomDataRoundtrips
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestBackupRestoreRandomDataRoundtrips2382557330
    test_log_scope.go:80: use -show-logs to present logs inline
    panic.go:1038: -- test log scope end --
    panic.go:1038: geos: no locations to init GEOS
        goroutine 4118 [running]:
        runtime/debug.Stack()
        	GOROOT/src/runtime/debug/stack.go:24 +0x65
        github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func1()
        	github.com/cockroachdb/cockroach/pkg/util/leaktest/leaktest.go:110 +0x11f
        panic({0x402f040, 0xc00416f500})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc001ffc240, {0x55dbf00, 0xc00012e020})
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:548 +0x325
        panic({0x402f040, 0xc00416f500})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/sql/randgen.encodeInvertedIndexHistogramUpperBounds(0x7b0fa40, {0x56b26b8, 0xc004abbda0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:368 +0x467
        github.com/cockroachdb/cockroach/pkg/sql/randgen.randHistogram(0x555e5a0, 0x7b0fa40)
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:301 +0x14c
        github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator.func1(0xc00404c900)
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:220 +0x92
        github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator(0xc001b7cc60, {0xc003cd6d40, 0x4, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:251 +0xa4d
        github.com/cockroachdb/cockroach/pkg/sql/randgen.MultiStatementMutation.Mutate(0x90, 0x43eb22b, {0xc003cd6d40, 0x8e, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:97 +0x2e
        github.com/cockroachdb/cockroach/pkg/sql/randgen.RandCreateTables(0x14, {0x43eb22b, 0x5}, 0x4, {0xc004809640, 0x3, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/schema.go:85 +0x157
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTablesN(0xc004809670, 0xc0048096a0)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:87 +0x187
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTables(0x3213bad)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:74 +0x33
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.wrapCommonSetup.func1(0x3deb280)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:44 +0x22
        github.com/cockroachdb/cockroach/pkg/ccl/backupccl.TestBackupRestoreRandomDataRoundtrips(0xc0026ec4e0)
        	github.com/cockroachdb/cockroach/pkg/ccl/backupccl/backup_rand_test.go:53 +0x2cc
        testing.tRunner(0xc0026ec4e0, 0x4654290)
        	GOROOT/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	GOROOT/src/testing/testing.go:1306 +0x35a
--- FAIL: TestBackupRestoreRandomDataRoundtrips (0.20s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreRandomDataRoundtrips.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreRandomDataRoundtrips [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4011410&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4011410&tab=artifacts#/) on master @ [5ac733bb4927020bc1c52da24b2591742fde8e1f](https://github.com/cockroachdb/cockroach/commits/5ac733bb4927020bc1c52da24b2591742fde8e1f):


```
=== RUN   TestBackupRestoreRandomDataRoundtrips
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestBackupRestoreRandomDataRoundtrips2640192899
    test_log_scope.go:80: use -show-logs to present logs inline
    panic.go:1038: -- test log scope end --
    panic.go:1038: geos: no locations to init GEOS
        goroutine 4119 [running]:
        runtime/debug.Stack()
        	GOROOT/src/runtime/debug/stack.go:24 +0x65
        github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func1()
        	github.com/cockroachdb/cockroach/pkg/util/leaktest/leaktest.go:110 +0x11f
        panic({0x402f560, 0xc003a80ce0})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc00169a900, {0x55dc4e0, 0xc000136020})
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:548 +0x325
        panic({0x402f560, 0xc003a80ce0})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/sql/randgen.encodeInvertedIndexHistogramUpperBounds(0x7b10a40, {0x56b2cf8, 0xc004f2a750})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:368 +0x467
        github.com/cockroachdb/cockroach/pkg/sql/randgen.randHistogram(0x555ebc0, 0x7b10a40)
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:301 +0x14c
        github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator.func1(0xc00484d980)
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:220 +0x92
        github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator(0xc003072a20, {0xc0028e85a0, 0x5, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:251 +0xa4d
        github.com/cockroachdb/cockroach/pkg/sql/randgen.MultiStatementMutation.Mutate(0x90, 0x43eb74b, {0xc0028e85a0, 0x8e, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:97 +0x2e
        github.com/cockroachdb/cockroach/pkg/sql/randgen.RandCreateTables(0x14, {0x43eb74b, 0x5}, 0x5, {0xc005517640, 0x3, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/schema.go:85 +0x157
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTablesN(0xc005517670, 0xc0055176a0)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:87 +0x187
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTables(0x3213f2d)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:74 +0x33
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.wrapCommonSetup.func1(0x3deb700)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:44 +0x22
        github.com/cockroachdb/cockroach/pkg/ccl/backupccl.TestBackupRestoreRandomDataRoundtrips(0xc004118820)
        	github.com/cockroachdb/cockroach/pkg/ccl/backupccl/backup_rand_test.go:53 +0x2cc
        testing.tRunner(0xc004118820, 0x46547e0)
        	GOROOT/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	GOROOT/src/testing/testing.go:1306 +0x35a
--- FAIL: TestBackupRestoreRandomDataRoundtrips (0.24s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreRandomDataRoundtrips.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreRandomDataRoundtrips [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4017861&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4017861&tab=artifacts#/) on master @ [3471dc50db8406f3faa638940492dbe0cab53435](https://github.com/cockroachdb/cockroach/commits/3471dc50db8406f3faa638940492dbe0cab53435):


```
=== RUN   TestBackupRestoreRandomDataRoundtrips
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestBackupRestoreRandomDataRoundtrips2187492093
    test_log_scope.go:80: use -show-logs to present logs inline
    panic.go:1038: -- test log scope end --
    panic.go:1038: geos: no locations to init GEOS
        goroutine 4075 [running]:
        runtime/debug.Stack()
        	GOROOT/src/runtime/debug/stack.go:24 +0x65
        github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func1()
        	github.com/cockroachdb/cockroach/pkg/util/leaktest/leaktest.go:110 +0x11f
        panic({0x4022720, 0xc001854580})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc0017d62d0, {0x55cbc20, 0xc0001b0020})
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:548 +0x325
        panic({0x4022720, 0xc001854580})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/sql/randgen.encodeInvertedIndexHistogramUpperBounds(0x7af9aa0, {0x56a2038, 0xc0040f05a0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:368 +0x467
        github.com/cockroachdb/cockroach/pkg/sql/randgen.randHistogram(0x554e640, 0x7af9aa0)
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:301 +0x14c
        github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator.func1(0xc002cf3380)
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:220 +0x92
        github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator(0xc0016b8f30, {0xc002202420, 0x3, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:251 +0xa4d
        github.com/cockroachdb/cockroach/pkg/sql/randgen.MultiStatementMutation.Mutate(0x90, 0x43de28b, {0xc002202420, 0x8e, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:97 +0x2e
        github.com/cockroachdb/cockroach/pkg/sql/randgen.RandCreateTables(0x14, {0x43de28b, 0x5}, 0x3, {0xc004edf640, 0x3, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/schema.go:85 +0x157
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTablesN(0xc004edf670, 0xc004edf6a0)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:87 +0x187
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTables(0x3207f8d)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:74 +0x33
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.wrapCommonSetup.func1(0x3ddef40)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:44 +0x22
        github.com/cockroachdb/cockroach/pkg/ccl/backupccl.TestBackupRestoreRandomDataRoundtrips(0xc00393d1e0)
        	github.com/cockroachdb/cockroach/pkg/ccl/backupccl/backup_rand_test.go:53 +0x2cc
        testing.tRunner(0xc00393d1e0, 0x4645d28)
        	GOROOT/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	GOROOT/src/testing/testing.go:1306 +0x35a
--- FAIL: TestBackupRestoreRandomDataRoundtrips (0.31s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreRandomDataRoundtrips.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreRandomDataRoundtrips [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4027420&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4027420&tab=artifacts#/) on master @ [65c6377f4f486d537ef7dfc59f8f9c78f1a6018a](https://github.com/cockroachdb/cockroach/commits/65c6377f4f486d537ef7dfc59f8f9c78f1a6018a):


```
=== RUN   TestBackupRestoreRandomDataRoundtrips
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestBackupRestoreRandomDataRoundtrips1421503098
    test_log_scope.go:80: use -show-logs to present logs inline
    panic.go:1038: -- test log scope end --
    panic.go:1038: geos: no locations to init GEOS
        goroutine 4183 [running]:
        runtime/debug.Stack()
        	GOROOT/src/runtime/debug/stack.go:24 +0x65
        github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func1()
        	github.com/cockroachdb/cockroach/pkg/util/leaktest/leaktest.go:110 +0x11f
        panic({0x4022a20, 0xc002a63d60})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc0015d4240, {0x55cbf00, 0xc00007e038})
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:548 +0x325
        panic({0x4022a20, 0xc002a63d60})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/sql/randgen.encodeInvertedIndexHistogramUpperBounds(0x7af9aa0, {0x56a2318, 0xc002c15980})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:368 +0x467
        github.com/cockroachdb/cockroach/pkg/sql/randgen.randHistogram(0x554e920, 0x7af9aa0)
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:301 +0x14c
        github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator.func1(0xc0053a6f00)
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:220 +0x92
        github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator(0xc001e69080, {0xc0037f0000, 0x4, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:251 +0xa4d
        github.com/cockroachdb/cockroach/pkg/sql/randgen.MultiStatementMutation.Mutate(0x90, 0x43de5ab, {0xc0037f0000, 0x8e, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:97 +0x2e
        github.com/cockroachdb/cockroach/pkg/sql/randgen.RandCreateTables(0x14, {0x43de5ab, 0x5}, 0x4, {0xc003957640, 0x3, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/schema.go:85 +0x157
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTablesN(0xc003957670, 0xc0039576a0)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:87 +0x187
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTables(0x320824d)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:74 +0x33
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.wrapCommonSetup.func1(0x3ddf240)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:44 +0x22
        github.com/cockroachdb/cockroach/pkg/ccl/backupccl.TestBackupRestoreRandomDataRoundtrips(0xc002d32000)
        	github.com/cockroachdb/cockroach/pkg/ccl/backupccl/backup_rand_test.go:53 +0x2cc
        testing.tRunner(0xc002d32000, 0x4646060)
        	GOROOT/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	GOROOT/src/testing/testing.go:1306 +0x35a
--- FAIL: TestBackupRestoreRandomDataRoundtrips (0.17s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreRandomDataRoundtrips.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreRandomDataRoundtrips [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4033753&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4033753&tab=artifacts#/) on master @ [e3be114fc6fe8eda07aafbc7ee43cc452456f270](https://github.com/cockroachdb/cockroach/commits/e3be114fc6fe8eda07aafbc7ee43cc452456f270):


```
=== RUN   TestBackupRestoreRandomDataRoundtrips
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestBackupRestoreRandomDataRoundtrips2348699824
    test_log_scope.go:80: use -show-logs to present logs inline
    panic.go:1038: -- test log scope end --
    panic.go:1038: geos: no locations to init GEOS
        goroutine 4184 [running]:
        runtime/debug.Stack()
        	GOROOT/src/runtime/debug/stack.go:24 +0x65
        github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func1()
        	github.com/cockroachdb/cockroach/pkg/util/leaktest/leaktest.go:110 +0x11f
        panic({0x4027140, 0xc003f5c460})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc0020485a0, {0x55d1880, 0xc00012e020})
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:548 +0x325
        panic({0x4027140, 0xc003f5c460})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/sql/randgen.encodeInvertedIndexHistogramUpperBounds(0x7b01ac0, {0x56a7db8, 0xc00382d4d0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:368 +0x467
        github.com/cockroachdb/cockroach/pkg/sql/randgen.randHistogram(0x5554360, 0x7b01ac0)
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:301 +0x14c
        github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator.func1(0xc002d8e300)
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:220 +0x92
        github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator(0xc003a508a0, {0xc004758000, 0x5, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:251 +0xa4d
        github.com/cockroachdb/cockroach/pkg/sql/randgen.MultiStatementMutation.Mutate(0x90, 0x43e374b, {0xc004758000, 0x8e, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:97 +0x2e
        github.com/cockroachdb/cockroach/pkg/sql/randgen.RandCreateTables(0x14, {0x43e374b, 0x5}, 0x5, {0xc005d5f640, 0x3, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/schema.go:85 +0x157
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTablesN(0xc005d5f670, 0xc005d5f6a0)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:87 +0x187
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTables(0x320b8cd)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:74 +0x33
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.wrapCommonSetup.func1(0x3de3560)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:44 +0x22
        github.com/cockroachdb/cockroach/pkg/ccl/backupccl.TestBackupRestoreRandomDataRoundtrips(0xc002ea7860)
        	github.com/cockroachdb/cockroach/pkg/ccl/backupccl/backup_rand_test.go:53 +0x2cc
        testing.tRunner(0xc002ea7860, 0x464b438)
        	GOROOT/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	GOROOT/src/testing/testing.go:1306 +0x35a
--- FAIL: TestBackupRestoreRandomDataRoundtrips (0.17s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreRandomDataRoundtrips.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreRandomDataRoundtrips [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4070911&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4070911&tab=artifacts#/) on master @ [d66c14663e027a5d9699b8a15e3d6f5821877ef5](https://github.com/cockroachdb/cockroach/commits/d66c14663e027a5d9699b8a15e3d6f5821877ef5):


```
=== RUN   TestBackupRestoreRandomDataRoundtrips
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestBackupRestoreRandomDataRoundtrips572986525
    test_log_scope.go:80: use -show-logs to present logs inline
    panic.go:1038: -- test log scope end --
    panic.go:1038: geos: no locations to init GEOS
        goroutine 4262 [running]:
        runtime/debug.Stack()
        	GOROOT/src/runtime/debug/stack.go:24 +0x65
        github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func1()
        	github.com/cockroachdb/cockroach/pkg/util/leaktest/leaktest.go:110 +0x11f
        panic({0x408a400, 0xc003e68060})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc00171c3f0, {0x56d8820, 0xc00007e038})
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:548 +0x325
        panic({0x408a400, 0xc003e68060})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/sql/randgen.encodeInvertedIndexHistogramUpperBounds(0x7c29d40, {0x57af098, 0xc00494e930})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:368 +0x467
        github.com/cockroachdb/cockroach/pkg/sql/randgen.randHistogram(0x565af60, 0x7c29d40)
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:301 +0x14c
        github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator.func1(0xc0052a5800)
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:220 +0x92
        github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator(0xc002b8ca50, {0xc000a3c050, 0x5, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:251 +0xa4d
        github.com/cockroachdb/cockroach/pkg/sql/randgen.MultiStatementMutation.Mutate(0x90, 0x4448bb0, {0xc000a3c050, 0x8e, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:97 +0x2e
        github.com/cockroachdb/cockroach/pkg/sql/randgen.RandCreateTables(0x14, {0x4448bb0, 0x5}, 0x5, {0xc005765640, 0x3, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/schema.go:85 +0x157
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTablesN(0xc005765670, 0xc0057656a0)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:87 +0x187
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTables(0x326610d)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:74 +0x33
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.wrapCommonSetup.func1(0x3e458c0)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:44 +0x22
        github.com/cockroachdb/cockroach/pkg/ccl/backupccl.TestBackupRestoreRandomDataRoundtrips(0xc000a91860)
        	github.com/cockroachdb/cockroach/pkg/ccl/backupccl/backup_rand_test.go:53 +0x2cc
        testing.tRunner(0xc000a91860, 0x46b2908)
        	GOROOT/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	GOROOT/src/testing/testing.go:1306 +0x35a
--- FAIL: TestBackupRestoreRandomDataRoundtrips (0.19s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreRandomDataRoundtrips.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreRandomDataRoundtrips [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4075336&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4075336&tab=artifacts#/) on master @ [6ff6c77e7aebec2edc947c1ed99a1f7cb67e1a3c](https://github.com/cockroachdb/cockroach/commits/6ff6c77e7aebec2edc947c1ed99a1f7cb67e1a3c):


```
=== RUN   TestBackupRestoreRandomDataRoundtrips
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestBackupRestoreRandomDataRoundtrips1839726973
    test_log_scope.go:80: use -show-logs to present logs inline
    panic.go:1038: -- test log scope end --
    panic.go:1038: geos: no locations to init GEOS
        goroutine 4227 [running]:
        runtime/debug.Stack()
        	GOROOT/src/runtime/debug/stack.go:24 +0x65
        github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func1()
        	github.com/cockroachdb/cockroach/pkg/util/leaktest/leaktest.go:110 +0x11f
        panic({0x4089c20, 0xc00127bd20})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc0016006c0, {0x56d7f20, 0xc00007e038})
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:548 +0x325
        panic({0x4089c20, 0xc00127bd20})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/sql/randgen.encodeInvertedIndexHistogramUpperBounds(0x7c29d40, {0x57ae778, 0xc003bed050})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:368 +0x467
        github.com/cockroachdb/cockroach/pkg/sql/randgen.randHistogram(0x565a660, 0x7c29d40)
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:301 +0x14c
        github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator.func1(0xc004a6b980)
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:220 +0x92
        github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator(0xc003af49f0, {0xc001648000, 0x4, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:251 +0xa4d
        github.com/cockroachdb/cockroach/pkg/sql/randgen.MultiStatementMutation.Mutate(0x90, 0x44483d0, {0xc001648000, 0x8e, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:97 +0x2e
        github.com/cockroachdb/cockroach/pkg/sql/randgen.RandCreateTables(0x14, {0x44483d0, 0x5}, 0x4, {0xc005297640, 0x3, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/schema.go:85 +0x157
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTablesN(0xc005297670, 0xc0052976a0)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:87 +0x187
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTables(0x326610d)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:74 +0x33
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.wrapCommonSetup.func1(0x3e451a0)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:44 +0x22
        github.com/cockroachdb/cockroach/pkg/ccl/backupccl.TestBackupRestoreRandomDataRoundtrips(0xc001121040)
        	github.com/cockroachdb/cockroach/pkg/ccl/backupccl/backup_rand_test.go:53 +0x2cc
        testing.tRunner(0xc001121040, 0x46b2128)
        	GOROOT/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	GOROOT/src/testing/testing.go:1306 +0x35a
--- FAIL: TestBackupRestoreRandomDataRoundtrips (0.17s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreRandomDataRoundtrips.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/backupccl.TestBackupRestoreRandomDataRoundtrips [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4090887&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4090887&tab=artifacts#/) on master @ [fdd672fed84323279f7070127f36cdbdc5c26b03](https://github.com/cockroachdb/cockroach/commits/fdd672fed84323279f7070127f36cdbdc5c26b03):


```
=== RUN   TestBackupRestoreRandomDataRoundtrips
    test_log_scope.go:79: test logs captured to: /artifacts/tmp/_tmp/2b7e992b6b49d0a96a6be6a67faab5b1/logTestBackupRestoreRandomDataRoundtrips486642451
    test_log_scope.go:80: use -show-logs to present logs inline
    panic.go:1038: -- test log scope end --
    panic.go:1038: geos: no locations to init GEOS
        goroutine 4235 [running]:
        runtime/debug.Stack()
        	GOROOT/src/runtime/debug/stack.go:24 +0x65
        github.com/cockroachdb/cockroach/pkg/util/leaktest.AfterTest.func1()
        	github.com/cockroachdb/cockroach/pkg/util/leaktest/leaktest.go:110 +0x11f
        panic({0x40b3340, 0xc000465e40})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc001556b40, {0x56fe800, 0xc00007e038})
        	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:548 +0x325
        panic({0x40b3340, 0xc000465e40})
        	GOROOT/src/runtime/panic.go:1038 +0x215
        github.com/cockroachdb/cockroach/pkg/sql/randgen.encodeInvertedIndexHistogramUpperBounds(0x7c68d80, {0x57d59f8, 0xc00302d7a0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:369 +0x487
        github.com/cockroachdb/cockroach/pkg/sql/randgen.randHistogram(0x5680e60, 0x7c68d80)
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:302 +0x14c
        github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator.func1(0xc0048e3b00)
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:221 +0x92
        github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator(0xc002cf6990, {0xc003f77d10, 0x5, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:252 +0xa4d
        github.com/cockroachdb/cockroach/pkg/sql/randgen.MultiStatementMutation.Mutate(0x90, 0x44724f2, {0xc003f77d10, 0x8e, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:98 +0x2e
        github.com/cockroachdb/cockroach/pkg/sql/randgen.RandCreateTables(0x14, {0x44724f2, 0x5}, 0x5, {0xc005bef640, 0x3, 0x0})
        	github.com/cockroachdb/cockroach/pkg/sql/randgen/schema.go:85 +0x157
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTablesN(0xc005bef670, 0xc005bef6a0)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:87 +0x187
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTables(0x32988ed)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:74 +0x33
        github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.wrapCommonSetup.func1(0x3e6dfc0)
        	github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:44 +0x22
        github.com/cockroachdb/cockroach/pkg/ccl/backupccl.TestBackupRestoreRandomDataRoundtrips(0xc001e56ea0)
        	github.com/cockroachdb/cockroach/pkg/ccl/backupccl/backup_rand_test.go:53 +0x2cc
        testing.tRunner(0xc001e56ea0, 0x46dc758)
        	GOROOT/src/testing/testing.go:1259 +0x102
        created by testing.(*T).Run
        	GOROOT/src/testing/testing.go:1306 +0x35a
--- FAIL: TestBackupRestoreRandomDataRoundtrips (0.19s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
Parameters in this failure:

- TAGS=bazel,gss
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBackupRestoreRandomDataRoundtrips.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/backupccl/BUILD.bazel
+++ b/pkg/ccl/backupccl/BUILD.bazel
@@ -160,7 +160,7 @@ go_test(
         "split_and_scatter_processor_test.go",
         "system_schema_test.go",
     ],
-    data = glob(["testdata/**"]),
+    data = glob(["testdata/**"]) + ["@cockroach//c-deps:libgeos"],
     embed = [":backupccl"],
     deps = [
         "//pkg/base",

